README.md

```diff
@@ -112,3 +112,13 @@ Both the json and object forms contain all the formation outlined in the respons
 of the [Get Transcript](https://www.rev.ai/docs#operation/GetTranscriptById) endpoint
 when using the json response schema. While the text output is a string containing
 just the text of your transcript
+
+### Streaming audio
+
+Once you have a streaming client setup with a MediaConfig and access token, you can obtain a transcription generator of your audio.
```

Comment:
I would show how to do this and also include something about each of the functions `on_error`, `on_connected` and `on_close`